.travis.yml

```diff
@@ -22,16 +28,16 @@ branches:
 before_install:
   - pip install -U pip wheel setuptools
   - pip install --no-cache-dir -U -r requirements-test.txt
+  - pip install --no-cache-dir -U -r requirements.txt
```

Comment:
remove this, not needed. Dependencies will be installed when doing `python setup.py -q develop`